src/mame/konami/hornet.cpp

```diff
@@ -704,40 +753,38 @@ void hornet_state::sysreg_w(offs_t offset, uint8_t data)
 
 /*****************************************************************************/
 
-uint8_t hornet_state::comm_eeprom_r()
+uint8_t sscope2_state::comm_eeprom_r()
 {
 	uint8_t r = 0;
 	r |= (m_lan_eeprom->do_read() & 1) << 1;
 	r |= m_lan_ds2401->read() & 1;
 	return r;
 }
 
-void hornet_state::comm_eeprom_w(uint8_t data)
+void sscope2_state::comm_eeprom_w(uint8_t data)
 {
 	m_eepromout->write(data, 0xff);
-	m_lan_ds2401->write((data >> 4) & 1);
+	m_lan_ds2401->write(BIT(data, 4));
 }
 
-void hornet_state::comm1_w(offs_t offset, uint32_t data, uint32_t mem_mask)
+void sscope2_state::comm1_w(offs_t offset, uint32_t data, uint32_t mem_mask)
 {
-	printf("comm1_w: %08X, %08X, %08X\n", offset, data, mem_mask);
+	LOGCOMM("comm1_w: %08X, %08X, %08X\n", offset, data, mem_mask);
 }
 
-void hornet_state::comm_rombank_w(uint32_t data)
+void sscope2_state::comm_rombank_w(uint32_t data)
 {
-	int bank = data >> 24;
-	if (m_comm_board_rom.found())
-		m_comm_bank->set_entry(bank & 0x7f);
+	m_comm_bank->set_entry((data >> 24) & 0x7f);
 }
 
-uint32_t hornet_state::comm0_unk_r(offs_t offset, uint32_t mem_mask)
+uint32_t sscope2_state::comm0_unk_r(offs_t offset, uint32_t mem_mask)
 {
-//  printf("comm0_unk_r: %08X, %08X\n", offset, mem_mask);
+//  LOGCOMM("comm0_unk_r: %08X, %08X\n", offset, mem_mask);
```

Comment:
You don’t need to have these commented out now that they’re conditional and disabled by default.